pt-BR/mail/chrome/messenger/SearchDialog.dtd

```diff
@@ -57,8 +57,8 @@
 
 <!-- Thread Pane Tooltips -->
 <!ENTITY columnChooser.tooltip "Clique para selecionar quais colunas mostrar">
-<!ENTITY threadColumn.tooltip "Clique para agrupar pelo tópico"> 
-<!ENTITY fromColumn.tooltip "Click to sort by from">
+<!ENTITY threadColumn.tooltip "Clique para agrupar pelo tópico">
+<!ENTITY fromColumn.tooltip "Clique para ordenar pelo campo &quote;De&quote;">
```

Comment:
Agreed. I'll change to "**pelo** remetente" though for consistency with the other lines.